youtube_dl/extractor/rdsca.py

```diff
@@ -0,0 +1,50 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    parse_iso8601,
+    url_basename,
+)
+
+
+class RDScaIE(InfoExtractor):
+    IE_NAME = 'RDS.ca'
+    _VALID_URL = r'http://(?:www\.)?rds\.ca/videos/(?P<id>.*)'
```

Comment:
Looks like all the URLs satisfy the following format: http://www.rds.ca/videos/.../<display_id>.<id>. `_VALID_URL` should be fixed correspondingly or test with another URL schema should be provided.